src/histolab/slide.py

```diff
@@ -64,6 +90,30 @@ def __repr__(self):
 
     # ---public interface methods and properties---
 
+    @lazyproperty
+    def base_mpp(self) -> float:
+        """Get microns-per-pixel resolution at scan magnification."""
```

Comment:
Considering that this is a public property and the docstring becaome automatically part of the doc in read the docs i would add the return statement like
```
Returns
        -------
        bla bla
        """
```

Reply:
ok done.